src/App.tsx

```diff
@@ -117,7 +116,7 @@ function App(): ReactElement {
               <DockerNotDetected />
             </Route>
             <Route path={Path.HOME}>
-              {isDarwin() ? <DockerNotDetected /> : <Landing />}
```

Comment:
The platform check also needs to be removed in:
`eventHandler.ts` ('disconnect' case);
`DockerNotDetected`.